releasenotes/notes/fix-split-by-domain-requests-978e1f51a5b3f77d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    requests: fixes split by domain
```

Comment:
is there a better description we can use here? what was fixed?

Reply:
I updated the release note